src/functions-and-arrays.js

```diff
@@ -52,14 +160,38 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  if(arr.length == 0) return null;
+  let newArray = []
+  for(let i = 0; i < arr.length; i++){
+    if(!newArray.includes(arr[i])){
+      newArray.push(arr[i])
+    }
+
+  }
+  return newArray
+}
+
+uniquifyArray(wordsUnique)
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, word) {
+  if(arr.length == 0) return null;
+  for(let i = 0; i < arr.length; i++){
```

Comment:
Nice logic here. You also could return the method to simplify your answer.
Instead of the for loop. You can use the includes() method.

`return arr.includes(word);`

Keep in mind that you did it correct. It's just an easier way of doing it.